cloud-checksum/src/checksum/aws_etag.rs

```diff
@@ -5,12 +5,73 @@
 use crate::checksum::standard::StandardCtx;
 use crate::error::Error::ParseError;
 use crate::error::{Error, Result};
+use crate::io::Provider;
+use phf::phf_ordered_map;
 use std::cmp::Ordering;
 use std::fmt::{Display, Formatter};
 use std::hash::{Hash, Hasher};
 use std::str::FromStr;
 use std::sync::Arc;
 
+/// Defines the "best" order for part sizes that should be preferenced when copying/generating
+/// new checksums. This list takes into account defaults that are likely to show up in the AWS CLI
+/// and SDKs.
+pub static PART_SIZE_ORDERING: phf::OrderedMap<u64, u64> = phf_ordered_map! {
+    // AWS CLI/boto3 uses 8MiB as the default:
+    // https://github.com/aws/aws-cli/blob/b9459db122d9f596a4570b6b5ecca44311b48fc2/awscli/customizations/s3/transferconfig.py#L21
+    // https://github.com/boto/boto3/blob/0442d32d2d2cbc5efbe158f6336993d1ee89b36b/boto3/s3/transfer.py#L243
+    // Java V2 uses 8MiB:
+    // https://github.com/aws/aws-sdk-java-v2/blob/5463dd3d403450e167408895c152c62884da65bd/services/s3/src/main/java/software/amazon/awssdk/services/s3/internal/multipart/MultipartConfigurationResolver.java#L28
+    8388608_u64 => 0,
```

Comment:
is it possible these might be better defined with some form of constants

8 * MiB

(I presume rust lets us define some constants that are resolved in epxressions at compile time)

Reply:
I think I can expand what the macro does use constants.